### PR TITLE
Mesh remove isolated vertex

### PR DESCRIPTION
**Pull request by kerautret**
# PR Description

By default the Meshreader imports all vertex even if they are not associated to a face. This PR adds a method to remove them that allows to avoid errors when applying conversion to a SurfaceMesh for instance. It also include a read improvement to ignore path when the material path is wrong and load by default from the current path.

# Checklist

- [x] Unit-test of your feature with [Catch](http://dgtal.org/doc/stable/moduleCatch.html).
- [x] Doxygen documentation of the code completed (classes, methods, types, members...)
- [x] Documentation module page added or updated.
- [x] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtal/blob/master/ChangeLog.md) added.
- [x] No warning raised in Debug mode.
- [x] All continuous integration tests pass (Github Actions)

### CODE REVIEW

File: src/DGtal/shapes/Mesh.ih [hunk 1]
```diff
@@ -435,14 +437,62 @@ DGtal::Mesh<TPoint>::clearFaces(){
...
+        {
+            face[i]=translateIndexId[face[i]];
+        }
+    }
+}
+
+template<typename TPoint>
+inline
+void
+DGtal::Mesh<TPoint>::changeScale(const typename TPoint::Component aScale){
```
Comment (dcoeurjo) [inline src/DGtal/shapes/Mesh.ih:491]: The method name is not great... `rescale` ?

**Review by dcoeurjo — CHANGES_REQUESTED** [review]
Thanks for the PR. Just minor comment

File: src/DGtal/shapes/Mesh.ih (hunk 1 above)
Comment (kerautret) [reply src/DGtal/shapes/Mesh.ih:491]: sure ;)

**Comment by kerautret** [comment]
thanks @dcoeurjo just done, but perhaps DGtalTools will failed from the renaming I will do it afterwards

**Comment by kerautret** [comment]
Done small PR in DGtalTools : https://github.com/DGtal-team/DGtalTools/pull/453 that need to be merged before

**Pull request merged**